mk/check.mk

```diff
@@ -89,6 +89,15 @@ check: format lint ## Dev: Run code checks (go fmt, go vet, ...)
 	fi; \
 	if [ "$$FAILED" = true ]; then exit 1; fi
 
+.PHONY: check/rbac
+check/rbac:
+	@RBAC_CHANGED=$$(git --no-pager diff --name-only -- deployments/ | xargs grep -E 'kind: (Role|RoleBinding|ClusterRole|ClusterRoleBinding)' || true); \
```

Comment:
I'm not sure this will work, git diff only checks unstaged changed so if a person commits it without running the check first this won't fail. I think we should check changes against `base` branch (mostly `master`, but during the release it's `release-X`, but I think `master` would be good enough).

Reply:
```suggestion
	@RBAC_CHANGED=$$(git --no-pager diff --name-only master -- deployments/ | xargs grep -E 'kind: (Role|RoleBinding|ClusterRole|ClusterRoleBinding)' || true); \
```